metron-interface/metron-alerts/src/app/pcap/service/pcap.service.ts

```diff
@@ -56,8 +56,7 @@ export class PcapService {
     public getPackets(id: string, pageId: number): Observable<Pdml> {
         return this.http.get(`/api/v1/pcap/${id}/pdml?page=${pageId}`, new RequestOptions({headers: new Headers(this.defaultHeaders)}))
             .map(HttpUtil.extractData)
-            .catch(HttpUtil.handleError)
-            .onErrorResumeNext();
```

Comment:
What are the implications of removing `onErrorResumeNext()`?

Reply:
It won't catch the error if that line is included.

Reply:
So previously it would catch the error and continue on regardless. Now it catches the error and handles it?

Reply:
It was not passing the error on to the observable.